samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Mobile/Uno.Toolkit.Samples.Mobile.csproj

```diff
@@ -5,6 +5,7 @@
     <!-- <TargetFrameworks>$(TargetFrameworks);net7.0-macos</TargetFrameworks> -->
     <SingleProject>true</SingleProject>
     <OutputType>Exe</OutputType>
+    <ApplicationId>uno.platform.toolkit-uwp</ApplicationId>
```

Comment:
Same here?

Reply:
Yeah this is the non-WinUI sample app, right? So we designated it with the -uwp suffix in the app id

Reply:
@jeromelaban should remove the uwp suffix or not?

Reply:
Indeed, I missed that it's the uwp variant.